terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts

```diff
@@ -221,6 +221,18 @@ async function addSSMParameterRunnerConfig(
   console.debug(`Created SSM Parameters(s): ${createdSSMParams.join(',')}`);
 }
 
+function getLaunchTemplateName(runnerParameters: RunnerInputParameters): Array<string | undefined> {
```

Comment:
nit: returning an object with two named fields, or a tuple type `[string | undefined, string | undefined]` is more suited here.